integration-tests/integration/features/case-list/case-list.spec.js

```diff
@@ -21,10 +21,10 @@ And('I should see sub navigation with default dates', () => {
   cy.get('.pac-pagination-link--text').eq(0).contains('Next day')
 })
 
-And('I should see that {int} defendants have possible nDelius records', $int => {
+And('I should see that {int} defendant have possible nDelius records', ($int, $string) => {
```

Comment:
The `$string` parameter is never set in the step template (see suggestion for what I think it should be)- because of the other comment I left about the feature file I don't think this code is ever getting called which is why it hasn't caused any errors.

```suggestion
And('I should see that {int} {string} have possible nDelius records', ($int, $string) => {
```

Reply:
👍